Core/clim-core/commands/commands.lisp

```diff
@@ -74,8 +74,10 @@
 (defun menu-items-from-list (menu)
   (mapcar
    #'(lambda (item)
-       (destructuring-bind (name type value &rest args) item
-         (apply #'make-menu-item name type value args)))
+       (if (atom item)
+           item
+           (destructuring-bind (name type value &rest args) item
+             (apply #'make-menu-item name type value args))))
```

Comment:
Please exercise this in one of the examples/tests.

Reply:
fixed (extended menu-test)